src/pages/FlssPrototypes/SampleLoanDisplay.vue

```diff
@@ -0,0 +1,1282 @@
+<template>
+	<www-page-corporate
```

Comment:
Please use the `www-page-minimal` here which will still require the theme attributes but not need the corporate-logo-url...